samples/EventImporter/Provider.Tests/TestStartup.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using PactNet.AspNetCore.Messaging;
+using PactNet.AspNetCore.Messaging.Options;
+
+namespace Provider.Tests
+{
+    public class TestStartup
+    {
+        private readonly Startup inner;
+        private readonly IConfiguration configuration;
+
+        public TestStartup(IConfiguration configuration)
+        {
+            this.configuration = configuration;
+
+            this.inner = new Startup(configuration);
+        }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.Configure<MessagingVerifierOptions>(configuration.GetSection("PactMessageMiddleware"));
```

Comment:
This should probably be a dedicated extension method like `services.AddPactMessaging(options => { ... })`;

Reply:
This is linked to the sample app. I'll add it in another PR with the middleware unit tests. The new one should also contain the other middleware like providerstate, auth, etc.